kvm-forum-hackfest-2017/README.md

```diff
@@ -0,0 +1,41 @@
+# KVM Forum KubeVirt Workshop 2017
+Have a room at KVM Forum KubeVirt Workshop 2017 to allow people interested in
+KubeVirt to sit together and talk.
+There will be a few sessions to have a little guidance/structure.
```

Comment:
Perfect - I actually also considered 23/24 and added them as suggestions to the doc.

We can shuffle the topics around to make sure that relevant stakeholders can participate for certain topics.